src/js/controller/model.js

```diff
@@ -256,6 +260,9 @@ define([
                 this.playbackComplete();
             }
 
+            // Restore the playback rate to the provider in case it changed while detached and we reused a video tag.
+            this.setPlaybackRate(this.get('playbackRate'));
```

Comment:
I would like to maintain the `defaultPlaybackRate` property on the video tag, however that property ends up getting cleared out in a number of circumstances.  For example, DASH really does a number on the video tag's properties since it will reset a lot of them between playlist items. Because of that the provider is maintaining the playback rate property to reset it later.  This sort of logic could be done in provider on attach and detach and this can just query the provider for the playback rate that's expected to be used.